common/constants.go

```diff
@@ -82,7 +82,8 @@ type (
 )
 
 // MaxTaskTimeout is maximum task timeout allowed. 366 days in seconds
-const MaxTaskTimeout = 31622400
+const MaxTaskTimeout = MaxTaskTimeoutSeconds * time.Second
+const MaxTaskTimeoutSeconds = 31622400
```

Comment:
I would remove this.

Reply:
Will remove after we convert intermediate formats that need UnixNano (WorkflowExecutionInfo...)